cliboa/scenario/transform/file.py

```diff
@@ -773,3 +774,46 @@ def execute(self, *args):
                 raise InvalidParameter(
                     "'format' must set one of the followings [tar, zip]"
                 )
+
+class ExecuteShellScript(FileBaseTransform):
+    """
+    Hash All Column Value
+    """
+    def __init__(self):
+        super().__init__()
+        self._command = ""
+        self._work_dir = ""
+
+    def command(self, command):
+        self._command = command
+
+    def work_dir(self, work_dir):
+        self._work_dir = work_dir
+
+    def execute(self, *args):
+        # essential parameters check
+        valid = EssentialParameters(
+            self.__class__.__name__,
+            [
+                self._command
+            ],
+        )
+        valid()
+
+        # Setting Up Directory
+        default_dir =os.getcwd()
+
+        if self._work_dir:
+            os.chdir(self._work_dir)
+
```

Comment:
https://github.com/BrainPad/cliboa/blob/master/cliboa/scenario/base.py#L128-L146
There is a method that is almost exactly the same with your code.
Please use this.
Uage ex:
```
r = clz._source_path_reader({"file": r"/resource/input/sample.txt" })
with open (r, mode="r") as f:
    print(f.read())`
```

ex2: 
```
r = clz._source_path_reader({"content": "select * from FOO" })
with open (r, mode="r") as f:
    print(f.read())`
```

Reply:
@bo-naoki-okamoto 
Thank you for the review.
Correct me if I'm wrong, that method change content to tmp file and return the file path.
to execute file the temp file must be sh and ideally have `#!/bin/s` In the head

So I dont think I can use that method

Reply:
Oh I see
you are right.

OK, please ignore this comment